tasks/clients.yml

```diff
@@ -13,8 +13,8 @@
 
 - name: Generate Server keys
   command: "{{ openvpn_etcdir }}/build-server.sh"
-  args:
-    creates: "{{ openvpn_keydir }}"
+#  args:
```

Comment:
This argument allows to have idempotence in the use of the `command` module, it shouldn't be removed. In any case, if this is because `openvpn_etcdir` is not the creation path, the variable should be corrected.